### DISCUSSION TITLE
Pimoroni RP2040 micropython

### DISCUSSION THREAD

**Discussion by Flipje1955** [micropython/micropython#10437] [RP2040 / Pico] [answered]
my collection of Pico's includes two Pimoroni RP2040 TINY's with 8M flash, bought for any future use of the indeed tiny size and/or the amount of flash. Now seeing a possible use for that storage, I now see that the board uses its own line of micropython software. 
Their most recent release is labelled 1.19.10 (Nov 18, 2022) which raises the impression that they are aligned to what all of you are developing and maintaing (big complements for all your efforts !!). Jjust to be sure, there is indeed no Pimoroni port available or foreseen ?

**Comment by robert-hh** [comment, marked as answer]
There is firmware available for your board at the Micropython download pages. Just grab the most recent daily build.

**Reply by Flipje1955** [reply]
noted I did not yet close this. Thx a lot for the quick pointer.
